src/components/pushClient/Banner.jsx

```diff
@@ -39,35 +39,38 @@ class BannerClient extends Component {
 
     return (
       <div className={styles['coz-banner-client']}>
-        <a
+        <ButtonLink
           href={t(mobileLink)}
-          target="_blank"
+          target
           className={styles['coz-btn-clientMobile']}
           onClick={e => {
             this.markAsSeen('banner')
           }}
         >
           <span>{t('Nav.btn-client-mobile')}</span>
-        </a>
+        </ButtonLink>
         <p className={styles['coz-banner-text']}>
           <span>{t('Nav.banner-txt-client')}</span>
-          <a
+          <ButtonLink
             href={t(desktopLink)}
-            target="_blank"
-            className={classNames(styles['c-btn'])}
+            target
```

Comment:
I think this prop has the wrong name, it should be `blank` or something like that. It's not very explicit what `target` does, and technically there are other values than `_blank` even though they are rarely used.

Reply:
None of the `target` values should be used actually. And besides `_blank`, every others are kinda useful only with frameset which is never. 
Anyway I asked myself the question of the naming but I thought that `target` speaks more for a link than `blank`, but maybe I'm wrong.